modules/apps/frontend-data-set/frontend-data-set-impl/src/main/java/com/liferay/frontend/data/set/internal/action/CustomFDSBulkActionListSerializerImpl.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.data.set.internal.action;
+
+import com.liferay.frontend.data.set.action.FDSBulkActionListSerializer;
+import com.liferay.frontend.data.set.constants.FDSTypes;
+import com.liferay.frontend.data.set.model.FDSActionDropdownItem;
+
+import java.util.Collections;
+import java.util.List;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Daniel Sanz
+ */
+@Component(
+	property = "dataset.type=" + FDSTypes.CUSTOM,
```

Comment:
Name of the property does not seem right. Maybe `fds.entry.type`? To be consistent with constants, `FDSTypes` should probably be `FDSEntryTypeConstants`.